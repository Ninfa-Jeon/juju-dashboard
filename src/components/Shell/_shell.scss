@import "vanilla-framework/scss/settings_colors";

.shell {
  background: $color-dark;
  color: $color-x-light;
  padding: 0.25rem 1rem 0;
  margin-right: -1rem;
  margin-left: -1rem;
  position: fixed;
  bottom: 0;
  width: 100%;
}
