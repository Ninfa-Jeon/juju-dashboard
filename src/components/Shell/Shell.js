import React, { Component } from "react";

import "./_shell.scss";

export default class Shell extends Component {
  render() {
    return (
      <div className="shell">
        <p>Welcome to the Juju shell - see the docs at https://jaas.ai/docs</p>
        <p>shell:-$</p>
      </div>
    );
  }
}
