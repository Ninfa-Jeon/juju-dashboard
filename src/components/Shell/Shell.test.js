import React from "react";
import { shallow } from "enzyme";

import Shell from "./Shell";

describe("Shell", () => {
  it("renders without crashing and matches snapshot", () => {
    const wrapper = shallow(<Shell />);
    expect(wrapper).toMatchSnapshot();
  });
});
