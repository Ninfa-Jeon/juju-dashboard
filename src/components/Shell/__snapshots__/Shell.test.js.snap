// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shell renders without crashing and matches snapshot 1`] = `
<div>
  <p>
    Welcome to the Juju shell - see the docs at https://jaas.ai/docs
  </p>
  <p>
    shell:-$
  </p>
</div>
`;
